packages/config/config/devices/0x0063/ge_46202.json

```diff
@@ -1,42 +1,63 @@
 // Jasco Products GE 46202
 // GE Enbrighten Z-Wave Plus Smart Switch
 {
-	"manufacturer": "Jasco Products",
-	"manufacturerId": "0x0063",
-	"label": "GE 46202",
-	"description": "GE Enbrighten Z-Wave Plus Smart Switch",
-	"devices": [
-		{
-			"productType": "0x4952",
-			"productId": "0x3137"
-		}
-	],
-	"firmwareVersion": {
-		"min": "0.0",
-		"max": "255.255"
+    "_approved": true,
+    "manufacturer": "Jasco Products",
+    "manufacturerId": "0x0063",
+    "label": "GE 46202",
+    "description": "GE Enbrighten Z-Wave Plus Smart Switch",
+    "devices": [
+        {
+            "productType": "0x4952",
+            "productId": "0x3137"
+        }
+    ],
+    "firmwareVersion": {
+        "min": "0.0",
+        "max": "255.255"
+    },
+    "supportsZWavePlus": true,
+    "associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+  			"isLifeline": true
+ 			},			
+		"2": {			
+			"label": "Basic - Local Load",
+			"maxNodes": 5
+			},
+		"3": {
+			"label": "Basic - Double Tap", 
+			"maxNodes": 5,
+			"isLifeline": true // Association to Group 3 is required for Double Tap support
+			} 
+	},
```

Comment:
indent here and the next few lines doesn't match the rest of the file

Reply:
Fixed to match the rest at least